src/image_bios.cpp

```diff
@@ -203,7 +203,7 @@ void BIOSUpdater::doInstall(const fs::path& file)
     // NOTE: This process may take a lot of time and we want to show the
     //       progress from original flashcp output.
     printf("Writing %s to %s\n", file.filename().c_str(), mtd);
-    int rc = system(strfmt("flashcp -v %s %s", file.c_str(), mtd).c_str());
+    int rc = system(strfmt(FLASHCP_CMD " -v %s %s", file.c_str(), mtd).c_str());
```

Comment:
Looks your work is incomplete. Yesterday it was merged another PR to the master and it brought some additional calls for `flashcp` and two new external tools. :)
Could you please rebase this branch and fix them too?

Reply:
ok

Reply:
Please do it